DR2-1357 Add policy to check for IAM_Admin logins

MancunianSam opened the pull request:
This checks Cloudtrail logs for a log in with IAM_Admin_Role.

The script will deploy it to all enabled regions.
This has already been deployed.

MancunianSam committed:
DR2-1357 Add policy to check for IAM_Admin logins

This checks Cloudtrail logs for a log in with IAM_Admin_Role.

The script will deploy it to all enabled regions.
This has already been deployed.

kurtismash commented on custodian/scripts/deploy-custodian.sh:
```diff
@@ -47,6 +47,13 @@ echo "Deploying CloudTrail detect root user policy"
 python ../custodian/scripts/build-policy-yml.py --cost_centre "$COST_CENTRE" --environment "$ENVIRONMENT" --filepath "../custodian/policies/cloudtrail/detect-root-logins.yml" --owner "$OWNER" --slack_webhook "$SLACK_WEBHOOK" --to_address "$TO_ADDRESS" --sqs_region "$SES_REGION" --sqs_account "$SQS_ACCOUNT"
 custodian run -s logs --region="$ROOT_LOGIN_REGION" deploy.yml
 
+python ../custodian/scripts/build-policy-yml.py --cost_centre "$COST_CENTRE" --environment "$ENVIRONMENT" --filepath "../custodian/policies/cloudtrail/detect_assume_admin_role.yml" --owner "$OWNER" --slack_webhook "$SLACK_WEBHOOK" --to_address "$TO_ADDRESS" --sqs_region "$SES_REGION" --sqs_account "$SQS_ACCOUNT"
```
I like your JQ solution here.

Can this be nested in an IF like TDR's reference generator thing. Some of our services use this role for normal access which we wouldn't want to alert on.

MancunianSam replied:
It's a good point. I've put an array in `ALERT_ON_ADMIN_ROLE_PROJECTS` which just has DR2 in it at the moment but any project that wants it can just put theirs in the array and it should work.

MancunianSam committed:
Add an if statement.

There is an array ALERT_ON_ADMIN_ROLE_PROJECTS

If the project is in that array, it will deploy the lambdas to check for
IAM_Admin_Role logins.

kurtismash commented:
Final thing, can you add the new rule to the `README.md`.

MancunianSam committed:
Add rule to README
